hip.cabal

```diff
@@ -70,6 +70,7 @@ Library
                  , Graphics.Image.Processing.Filter
                  , Graphics.Image.Processing.Hough
                  , Graphics.Image.Processing.Ahe
+                 , Graphics.Image.Processing.Laplacian
```

Comment:
There is no module with that name included in this PR

Reply:
@khilanravani If you'd like to place filters in their own modules you could do that, eg. a laplacian could go in `Graphics.Image.Processing.Filter.Laplacian`, but IMHO there is no real need for that. It's up to you though.

Reply:
Yes sir, will correct it.